src/OrchardCore.Modules/OrchardCore.OpenId/Controllers/UserInfoController.cs

```diff
@@ -56,6 +56,64 @@ public async Task<IActionResult> Me()
 
             var claims = new JObject();
 
+            if (principal.HasClaim(OpenIdConnectConstants.Claims.Scope, OpenIdConnectConstants.Scopes.Profile))
+            {
+                var preferredUsername = principal.FindFirst(OpenIdConnectConstants.Claims.PreferredUsername)
+                                                 ?.Value;
+                if (!string.IsNullOrEmpty(preferredUsername))
+                {
+                    claims[OpenIdConnectConstants.Claims.PreferredUsername] = preferredUsername;
+                }
+
+                var name = principal.FindFirst(OpenIdConnectConstants.Claims.Name)?.Value ??
+                              principal.FindFirst(ClaimTypes.Name)?.Value;
+
+                if (!string.IsNullOrEmpty(name))
+                {
+                    claims[OpenIdConnectConstants.Claims.Name] = name;
+                }
+
+                var familyName = principal.FindFirst(OpenIdConnectConstants.Claims.FamilyName)?.Value ??
+                           principal.FindFirst(ClaimTypes.Surname)?.Value;
+
+                if (!string.IsNullOrEmpty(familyName))
+                {
+                    claims[OpenIdConnectConstants.Claims.FamilyName] = familyName;
+                }
+
+                var givenName = principal.FindFirst(OpenIdConnectConstants.Claims.GivenName)?.Value ??
+                           principal.FindFirst(ClaimTypes.GivenName)?.Value;
+
+                if (!string.IsNullOrEmpty(givenName))
+                {
+                    claims[OpenIdConnectConstants.Claims.GivenName] = givenName;
+                }
+
+                var middleName = principal.FindFirst(OpenIdConnectConstants.Claims.MiddleName)
+                                          ?.Value;
+
+                if (!string.IsNullOrEmpty(middleName))
+                {
+                    claims[OpenIdConnectConstants.Claims.MiddleName] = middleName;
+                }
+
+                var picture = principal.FindFirst(OpenIdConnectConstants.Claims.Picture)
+                                       ?.Value;
+
+                if (!string.IsNullOrEmpty(picture))
+                {
+                    claims[OpenIdConnectConstants.Claims.Picture] = picture;
+                }
+
+                var updatedAt = principal.FindFirst(OpenIdConnectConstants.Claims.UpdatedAt)
+                                       ?.Value;
+
+                if (!string.IsNullOrEmpty(updatedAt))
+                {
+                    claims[OpenIdConnectConstants.Claims.UpdatedAt] = updatedAt;
```

Comment:
Since the `updatedAt` variable is a string, it will be returned as a JSON string in the userinfo payload, which won't be standard-compliant as the OIDC spec requires that dates be formatted as JSON integers. Consider converting it to a long or decimal (with invariant culture) to ensure this doesn't happen.